packages/payment-detection/src/erc777/superfluid-retriever.ts

```diff
@@ -120,6 +120,11 @@ export class SuperFluidInfoRetriever {
         timestamp: streamEvents[index].timestamp,
       });
     }
+    const newLastParameters = paymentEvents[paymentEvents.length - 1].parameters;
```

Comment:
This constant name confuses me a bit, what does it represent?

Reply:
I'm just copying the parameters of the last payment event. I call them 'new' because I'm updating them in the next few lines and then overwriting the old params.